jupyter/trustyai/container/Dockerfile

```diff
@@ -0,0 +1,55 @@
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+ARG UBI_VERSION
+ARG PYTHON_VERSION
+ARG PYTHON_VERSION_LONG
+ARG RELEASE
+ARG DATE
+ARG CUDA
+
+LABEL name="workbench-images:${CUDA}jupyter-trustyai-${UBI_VERSION}-${PYTHON_VERSION}_${RELEASE}_${DATE}" \
+    summary="Data Science + TrustyAI with ${PYTHON_VERSION_LONG} based on ${UBI_VERSION}" \
+    description="Data Science + TrustyAI with ${PYTHON_VERSION_LONG} based on ${UBI_VERSION}" \
+    io.k8s.description="Data Science + TrustyAI with ${PYTHON_VERSION_LONG} based on ${UBI_VERSION} for ODH or RHODS" \
+    io.k8s.display-name="Data Science + TrustyAI with ${PYTHON_VERSION_LONG} based on ${UBI_VERSION}" \
+    authoritative-source-url="https://github.com/opendatahub-io-contrib/workbench-images" \
+    io.openshift.build.commit.ref="${RELEASE}" \
+    io.openshift.build.source-location="https://github.com/opendatahub-io-contrib/workbench-images/jupyter/trustyai" \
+    io.openshift.build.image="https://quay.io/opendatahub-contrib/workbench-images:${CUDA}jupyter-trustyai-${UBI_VERSION}-${PYTHON_VERSION}_${RELEASE}_${DATE}"
+
+USER 0
+
+RUN yum install -y yum-utils && \
+    yum-config-manager --enable crb && \
+    yum install -y https://download.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm && \
+    yum -y clean all --enablerepo='*'
```

Comment:
Is it this necessary? I find strange that you need to enable EPEL 🤔 
Is it because your are not making the assumption for the base image that is provided?

Reply:
good point, I think it makes sense to base the TrustyAI image on the _datascience_ one.

Reply:
As I discussed with @tteofili , there are also many checks to do on packages because as it is now TrustyAI fixes pretty old versions of packages in its requirements (matplotlib, pandas, pyarrow,...), which will conflict with more recent ones that are in the datascience image. It would be worth to check if those are hard requirements, meaning TrustyAI really needs them at those versions, or if they can be bumped. If not, then maybe you should start from minimal-ubi9-py39 instead of datascience.

Reply:
I would expect we can bump ~all version to the latest without any real issue, the library uses pretty stable public API for them 🤔 

Btw I'm fine doing it incrementally with another PR if we prefer to move fast with this, it was just a general comment because I find strange the usage of EPEL in this context :)

Reply:
Yeah, the epel enablement is totally useless in this context. Also, the image should be based on ubi9, not c9s. I already pointed this out to @tteofili .